exchange/utils_test.go

```diff
@@ -607,93 +607,97 @@ func TestCreateSanitizedImpExt(t *testing.T) {
 		{
 			description: "imp.ext",
 			givenImpExt: map[string]json.RawMessage{
-				"anyBidder": json.RawMessage(`"anyBidderValues"`),
-				"data":      json.RawMessage(`"anyData"`),
-				"context":   json.RawMessage(`"anyContext"`),
-				"skadn":     json.RawMessage(`"anySKAdNetwork"`),
-				"gpid":      json.RawMessage(`"anyGPID"`),
-				"tid":       json.RawMessage(`"anyTID"`),
+				"arbitraryField": json.RawMessage(`"arbitraryValue"`),
+				"data":           json.RawMessage(`"anyData"`),
+				"context":        json.RawMessage(`"anyContext"`),
+				"skadn":          json.RawMessage(`"anySKAdNetwork"`),
+				"gpid":           json.RawMessage(`"anyGPID"`),
+				"tid":            json.RawMessage(`"anyTID"`),
 			},
 			givenImpExtPrebid: map[string]json.RawMessage{},
 			expected: map[string]json.RawMessage{
-				"data":    json.RawMessage(`"anyData"`),
-				"context": json.RawMessage(`"anyContext"`),
-				"skadn":   json.RawMessage(`"anySKAdNetwork"`),
-				"gpid":    json.RawMessage(`"anyGPID"`),
-				"tid":     json.RawMessage(`"anyTID"`),
+				"arbitraryField": json.RawMessage(`"arbitraryValue"`),
+				"data":           json.RawMessage(`"anyData"`),
+				"context":        json.RawMessage(`"anyContext"`),
+				"skadn":          json.RawMessage(`"anySKAdNetwork"`),
+				"gpid":           json.RawMessage(`"anyGPID"`),
+				"tid":            json.RawMessage(`"anyTID"`),
 			},
 			expectedError: "",
 		},
 		{
 			description: "imp.ext + imp.ext.prebid - Prebid Bidder Only",
 			givenImpExt: map[string]json.RawMessage{
-				"anyBidder": json.RawMessage(`"anyBidderValues"`),
-				"prebid":    json.RawMessage(`"ignoredInFavorOfSeparatelyUnmarshalledImpExtPrebid"`),
-				"data":      json.RawMessage(`"anyData"`),
-				"context":   json.RawMessage(`"anyContext"`),
-				"skadn":     json.RawMessage(`"anySKAdNetwork"`),
-				"gpid":      json.RawMessage(`"anyGPID"`),
-				"tid":       json.RawMessage(`"anyTID"`),
+				"unknownBidder": json.RawMessage(`"anyUnknownBidderValues"`),
```

Comment:
As mentioned in an earlier [comment](https://github.com/prebid/prebid-server/pull/3878#discussion_r1844627064) I think it makes more sense if all of these test cases in `TestCreateSanitizedImpExt` with key `"unknownBidder"` use `"arbitraryField"` instead.

Reply:
this should be addressed now in `utils_test.go`